assemblies/assembly_about-rhdh.adoc

```diff
@@ -0,0 +1,20 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-11-04
+
+ifdef::context[:parent-context-of-about-rhdh: {context}]
+
+:_mod-docs-content-type: ASSEMBLY
+
+ifndef::context[]
+[id="about-rhdh"]
+endif::[]
+ifdef::context[]
+[id="about-rhdh_{context}"]
+endif::[]
+= About {product}
+
+:context: about-rhdh
+
+{product} is a fully supported, enterprise-grade, open developer platform that you can use to build developer portals. This platform contains a supported and opinionated framework that helps reduce the friction and frustration of developers while boosting productivity. {product} simplifies decision-making by providing a developer experience that presents a selection of internally approved tools, programming languages, and developer resources within a self-managed portal. As a developer, you can use {product} to experience a streamlined development environment. {product} is driven by a centralized software catalog, providing efficiency to your microservices and infrastructure. It enables your product team to deliver quality code without any compromises.
+
+include::modules/discover/con_benefits-of-rhdh.adoc[leveloffset=+1]
```

Comment:
For consistency with the project, consider renaming the file con_benefits-of-rhdh.adoc into con-benefits-of-rhdh.adoc

```suggestion
include::modules/discover/con_benefits-of-rhdh.adoc[leveloffset=+1]
```